behave_restful/_utils.py

```diff
@@ -28,11 +28,8 @@ def load_module(module_name):
     :param str module_name:
         Name of the module to be imported as a string. 
     """
-    module_file = None
-    try:
-        module_file, pathname, description = imp.find_module(module_name)
-        module = imp.load_module(module_name, module_file, pathname, description)
-        return module
-    finally:
-        if module_file:
-            module_file.close()
+    module_spec = importlib.util.find_spec(module_name)
```

Comment:
I believe the problem is here. You're trying to find the spec and then use it to load the module. You just have to call `importlib.load_module(module_name)` and return the module. You don't need to raise any exception since it will raise the exception itself.

Reply:
Thanks, changed to [importlib.import_module](https://docs.python.org/3/library/importlib.html#importlib.import_module)(module_name).